epochtalk/epochtalk
# Themes

Test Procedure:

1) Visit Admin -> Settings -> Theme Page
2) Modify theme
3) Test preview button, ensure theme is only changed locally
4) Test save function, ensure theme is changed globally
5) Test reset function, should reset values on the form to current globally set theme
6) Test revert function, should reset global theme to default epochtalk theme.
7) Test permissions for theming by modifying permissions in manage roles page.
